Tasks/KubeloginInstallerV0/utils.ts

```diff
@@ -134,6 +148,42 @@ export async function unzipRelease(zipPath: string): Promise<string> {
   }
 }
 
+function getGithubEndPointToken(): string {
+  const githubEndpoint = taskLib.getInput("gitHubConnection", false);
+  const githubEndpointObject = taskLib.getEndpointAuthorization(githubEndpoint, true);
+  let githubEndpointToken: string = null;
+
+  if (!githubEndpointObject) {
+    throw new GitHubEndpointError(taskLib.loc("Failed to retrieve GitHub endpoint object."));
+  }
+  taskLib.debug("Endpoint scheme: " + githubEndpointObject.scheme);
+
+  switch (githubEndpointObject.scheme) {
+    case 'PersonalAccessToken':
+      githubEndpointToken = githubEndpointObject.parameters.accessToken;
+      break;
+    case 'OAuth':
+      githubEndpointToken = githubEndpointObject.parameters.accessToken;
+      break;
+    case 'Token':
+      githubEndpointToken = githubEndpointObject.parameters.accessToken;
+      break;
+    default:
+      throw new Error(
```

Comment:
Same for "Error" class. I recommend replacing all Error classes in this file to child-classes with meaningful name

Reply:
Added the error class